### PR TITLE
nothing

### PR DESCRIPTION
**Pull request by Coral-Reefs**
nothing

### CODE REVIEW

File: home.scss [hunk 1]
```diff
@@ -6,7 +6,7 @@ body {
     justify-content: center;
     flex-flow: column;
     text-align: center;
-    background: linear-gradient(to right top, #f4bfbf, #f8cbc0, #fad7c4, #fae4cc, #faf0d7);
+    background-image: linear-gradient(to right top, #f4bfbf, #f8cbc0, #fad7c4, #fae4cc, #faf0d7);
```
Comment (TheAmityD) [inline home.scss:9]: I deleted this "-image"... e.

**Review by ammmyd — COMMENTED** [review]
Pretty good on the home.css, I'll convert to SCSS.

**Review by ammmyd — APPROVED** [review]
Pretty good on the table, too

**Comment by ammmyd** [comment]
No.

**Pull request merged**